system/scripts/items/collections.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Melia.Shared.L10N;
+using Melia.Shared.Scripting;
+using Melia.Shared.Tos.Const;
+using Melia.Shared.World;
+using Melia.Zone;
+using Melia.Zone.Events;
+using Melia.Zone.Scripting;
+using Melia.Zone.World.Actors.Characters;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Melia.Zone.World.Actors.Monsters;
+using Melia.Zone.World.Items;
+using Melia.Zone.Network;
+using Yggdrasil.Util;
+
+public class CollectionItemScripts : GeneralScript
+{
+	[ScriptableFunction]
+	public ItemUseResult SCR_USE_COLLECTION_GET(Character character, Item item, string strArg, float numArg1, float numArg2)
+	{
+		var collectionName = strArg;
+
+		if (ZoneServer.Instance.Data.CollectionDb.Lookup(strArg, out var collectionData))
+		{
+			if (character.Collections.Add(collectionData.Id))
+			{
+				Send.ZC_SYSTEM_MSG(character, 119006);
```

Comment:
You might want to use `Character.SystemMessage` instead, which takes the class name of the message instead of the id, which is more future proof.

Reply:
Good catch on that, I've updated it.